_extras/checkout.md

```diff
@@ -153,113 +138,94 @@ The lessons you may submit to are:
 
 During the session,
-you are expected to ask questions about issues you noticed while reading your chosen lesson,
-and about our operations in general.
-If the discussion leader feels that you have adequately prepared,
-she will notify the training coordinator
-and ask you to sign up for a demonstration lesson (described in the next session).
-If she feels you have not prepared,
-she may ask you to come back for another session later.
-
-### Rubric
-<a name="rubric"></a>
-
-Trainees are expected to ask questions during their discussion session
-to show that they've thought about what they're going to teach,
-and about workshops in general.
+you are expected to ask questions to show that you've thought about what you're going to teach,
+and about how our workshops are run in general.
+
 Sample questions include:
 
 *   What were the most interesting or surprising things existing instructors learned
-    when they attended or taught their first workshop?
+when they attended or taught their first workshop?
 *   What pre- and during-workshop responsibilities did each instructor have?
 *   What lessons did people teach?
 *   How did instructors test installation of the required software?
 *   How long did lessons actually take to teach?
-    (Alternatively, how far did instructors actually get?)
+(Alternatively, how far did instructors actually get?)
 *   Did people adapt or modify the lesson material?
-    If so, how and why?
+If so, how and why?
 *   Where in the material are learners most likely to misunderstand things or get lost,
-    and why?
+and why?
 
 Trainees may also ask about:
 
 *   material in the lessons that they don't understand,
 *   how teaching these workshops has helped people's own work or careers, and
 *   anything else that comes to mind.
 
+If the discussion leader feels that you have adequately participated in the conversation,
+she will notify the training coordinator 
+who will then ask you to sign up for a teaching demonstration (described in the next session).
+
 ## Part 3: Teach a Short Demonstration Lesson
 <a name="demo-lesson"></a>
 
 Once you have completed your discussion session,
-the final step is to sign up for a 5-minute online demonstration lesson
+the final step is to sign up for a 5-minute teaching demonstration online using
 [this Etherpad]({{page.demopad}})
 (or add yourself to the top of the pad if none of the available times work for you).
-Like the discussion sessions,
-these demonstration lessons are run as video conferences using Google Hangouts,
-and the instructor leading the session will post connection details and other information
-in [the Etherpad]({{page.demopad}})
-shortly before it begins.
-
-For your teaching demonstration,
-will ask you to teach a short segment from your chosen lesson.
-If you want to do your final demonstration(s) in a language other than English,
+These demonstrations are run as video conferences using Google Hangouts. 
```

Comment:
The teaching demos are currently run with Google Hangouts (the discussion sessions are run with BlueJeans). I left this in because some folks may not have used hangouts before (I had trouble with it the first few times) and may want advanced warning to set it up. If the connection details are posted a day in advance, I can justify taking this out, but not if they're posted just a few minutes ahead of time. @gvwilson Can you let me know what the timeframe usually is?